README.md

```diff
@@ -1,9 +1,15 @@
 # Filehound
 
-Flexible and fluent interface for searching the file system
+[![Build Status](https://travis-ci.org/nspragg/filehound.svg)](https://travis-ci.org/nspragg/filehound)
+
```

Comment:
Once merged head to https://travis-ci.org/nspragg/filehound and enable Travis for the project. Should then start running the tests.